CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 **Added**:
 
+- **decidim-proposals**: Add authorization workflow to amend action[#5184](https://github.com/decidim/decidim/pull/5184)
```

Comment:
perhaps its more understandable, to describe it as:  Add :amend action to proposal's authorization workflow